sos/plugins/ceph.py

```diff
@@ -50,23 +50,41 @@ def setup(self):
         ])
 
         self.add_cmd_output([
-            "ceph status",
-            "ceph health detail",
-            "ceph osd tree",
-            "ceph osd stat",
-            "ceph osd dump",
             "ceph mon stat",
-            "ceph mon dump",
-            "ceph df",
+            "ceph-disk list",
+            "ceph osd erasure-code-profile ls",
+            "ceph -v"
+        ])
+
+        ceph_cmds = [
+            "status",
+            "health detail",
+            "osd tree",
+            "osd stat",
+            "osd df tree",
+            "osd dump",
+            "osd df",
+            "mon dump",
+            "df",
+            "df detail",
+            "fs ls",
+            "fs dump",
+            "pg dump",
+            "-s"
```

Comment:
@zerodayz the 'ceph -s' and 'ceph status' are same so you can remove one of them.